### ISSUE TITLE
extern_prelude error in contrib/lib/src/database.rs

### ISSUE THREAD

**Issue by sackery** [SergioBenitez/Rocket#740]
```rust
error[E0658]: access to extern crates through prelude is experimental (see issue #44660)
   --> /home/sackery/.cargo/git/checkouts/rocket-8bf16d9ca7e90bdc/fe9fad3/contrib/lib/src/databases.rs:709:19
    |
709 | impl Poolable for diesel::PgConnection {
    |                   ^^^^^^
    |
    = help: add #![feature(extern_prelude)] to the crate attributes to enable

error[E0658]: access to extern crates through prelude is experimental (see issue #44660)
   --> /home/sackery/.cargo/git/checkouts/rocket-8bf16d9ca7e90bdc/fe9fad3/contrib/lib/src/databases.rs:710:20
    |
710 |     type Manager = diesel::r2d2::ConnectionManager<diesel::PgConnection>;
    |                    ^^^^^^
    |
    = help: add #![feature(extern_prelude)] to the crate attributes to enable

error[E0658]: access to extern crates through prelude is experimental (see issue #44660)
   --> /home/sackery/.cargo/git/checkouts/rocket-8bf16d9ca7e90bdc/fe9fad3/contrib/lib/src/databases.rs:710:52
    |
710 |     type Manager = diesel::r2d2::ConnectionManager<diesel::PgConnection>;
    |                                                    ^^^^^^
    |
    = help: add #![feature(extern_prelude)] to the crate attributes to enable

error[E0658]: access to extern crates through prelude is experimental (see issue #44660)
   --> /home/sackery/.cargo/git/checkouts/rocket-8bf16d9ca7e90bdc/fe9fad3/contrib/lib/src/databases.rs:714:23
    |
714 |         let manager = diesel::r2d2::ConnectionManager::new(config.url);
    |                       ^^^^^^
    |
    = help: add #![feature(extern_prelude)] to the crate attributes to enable

error: aborting due to 4 previous errors
```

**Comment by jebrosen** [comment]
The feature names in Cargo.toml don't match the feature names in the code. It looks like #739 looks like it will fix this.

**Comment by SergioBenitez** [comment]
Should be fixed by a62248f7ce.
